_posts/2018-07-31-how-to-move-to-https.md

```diff
@@ -0,0 +1,100 @@
+
+![mixed content browser errors]({{ site.url }}/assets/img/blog/mixed-content-errors.png)
+
+In the case of an active asset that is compromised, if for example, it was a javascript file, an attacker can take full control of the page and any interaction you may have with it like entering passwords, or credit card information for example. The mechanism behind this is a [somewhat sophisticated man-in-the-middle attack](https://developers.google.com/web/fundamentals/security/prevent-mixed-content/what-is-mixed-content#mixed_content_types_security_threats_associated), but suffice it to say, if the browser recognizes the vulnerability, the [user experience we discussed in part one](/2018/06/26/why-you-need-https.html#https-improves-credibility-and-ux) will be poor in the best of scenarios, and total data compromise in the worst. Your audience, and by association your organization, will be seeing red.
+<br> ![Chrome misconfigured HTTPS]({{ site.url }}/assets/img/blog/misconfigured-https.png)
+
+### The good news about moving to HTTPS
+
+Ensuring your website serves assets exclusively over HTTPS is not as hard as it used to be, and is getting easier by the day.
+
+#### There are free digital certificates
+
+There's no such thing as a free lunch, but a free certificate from a reputable CA? It would seem so. People are just giving them away these days… Seriously, many years in the making since its founding by two Mozilla employees in 2012, the Let's Encrypt project has vowed to make the web a secure space and has successfully endeavored to become a trusted CA that literally does not charge for the certificates they provide. They have shorter lease cycles of 60 to 90 days, but they also offer tooling around automating the process of reinstalling newly provided certificates.
+
+#### There are easier and cheaper ways to install certificates
+
+With the advent of the aforementioned free certificate, many platform-as-a-service (PaaS) hosting options have incorporated low cost or free installation of certificates through their hosting platform sometimes as easily as a few clicks. Let's Encrypt has been [ adopted across a broad range of website hosting providers](https://community.letsencrypt.org/t/web-hosting-who-support-lets-encrypt/6920) like Squarespace, GitHub Pages, Dreamhost all of which we use alongside many others.
+
```

Comment:
"Let's Encrypt has been adopted across a broad range of website hosting providers like Squarespace, Github Pages, and Dreamhost, all of which we use alongside many others."